data/mock/pubkeyConverterStub.go

```diff
@@ -26,12 +26,12 @@ func (pcs *PubkeyConverterStub) Decode(humanReadable string) ([]byte, error) {
 }
 
 // Encode -
-func (pcs *PubkeyConverterStub) Encode(pkBytes []byte) string {
+func (pcs *PubkeyConverterStub) Encode(pkBytes []byte) (string, error) {
 	if pcs.EncodeCalled != nil {
-		return pcs.EncodeCalled(pkBytes)
+		return pcs.EncodeCalled(pkBytes), nil
```

Comment:
L7 should be `Encode(pkBytes []byte) (string, error)`, this way L31 becomes `return pcs.EncodeCalled(pkBytes)`

Reply:
missed it :facepalm: